NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+<packageSources>
+ <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+ <add key="Roslyn nightly builds" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
Going for the lib folder approach. Do you know what we have to do in csproj to include these files in the NuGet package?